.github/workflows/actions/quarto-dev/action.yml

```diff
@@ -3,23 +3,21 @@ description: 'Configures the image for Quarto Development'
 
 runs:
   using: "composite"
-  steps:          
+  steps:
       - name: Set Path
         shell: bash
         run: |
           echo "$HOME/bin" >> $GITHUB_PATH
-      
+
       - name: Configure Quarto
         shell: bash
         run: |
-          ./configure-linux.sh
-          
+          ./configure.sh
+          package/src/quarto-bld prepare-dist
```

Comment:
I'm not sure yet what happened here. I don't think this should be necessary, but without it the vendor folder is not copied over as part of the configure script, so lots of stuff is broken.

Reply:
@dragonstyle Is there a better or alternate way to do this?